packages/blade/src/tokens/global/typography.ts

```diff
@@ -0,0 +1,101 @@
+import type { FontFamily } from './fontFamily';
+import fontFamily from './fontFamily';
+
+type FontWeight = {
+  regular: 400;
+  bold: 700;
+};
+
+type FontSize = {
+  10: number;
```

Comment:
Why such a scale for fontSize? RFC propsed something else https://github.com/razorpay/blade/blob/master/rfcs/2021-19-02-units-for-typography-and-layout.md#how-will-we-store-and-use-it

Reply:
such a scale in the sense? RFC proposed the same thing na `font-size-10` and so on?

Reply:
RFC says 1,2,3,4,... but why are we going ahead with 10, 25, 50, 75, 100?
<img width="260" alt="Screenshot 2021-05-20 at 12 10 44 PM" src="https://user-images.githubusercontent.com/24487274/118931267-68242700-b964-11eb-82b7-4ebbaed9b29d.png">

Reply:
fontSizes till `1000` 🤔

Reply:
+1

Reply:
RFC was just taking an example. Refer [this RFC](https://github.com/razorpay/blade/blob/master/rfcs/2021-01-05-typography-scaling.md)

Reply:
We are doing 10, 25 etc because that's a general convention when talking about font scales and color scales we follow 100-900 and then increment/decrement based on the requirement. For more context [read this](https://graphicdesign.stackexchange.com/a/126061)